lineman/app/models/base_model.coffee

```diff
@@ -17,6 +17,13 @@ angular.module('loomioApp').factory 'BaseModel', ->
     initialize: (data) ->
       @updateFromJSON(data)
 
+    clone: ->
+      attrs = @baseSerialize()[@constructor.singular] or {}
+      clone = @recordsInterface.initialize attrs
```

Comment:
I don't trust this. BaseRecordsInterface:initialize inserts into the colleciton. I think you want to be calling new Model.. so maybe something like new @constructor(attrs)